notesapi/v1/tests/test_views.py

```diff
@@ -47,51 +44,66 @@ def setUp(self):
             ],
         }
 
-        self.expected_note = {
-            "created": "2014-11-26T00:00:00+00:00",
-            "updated": "2014-11-26T00:00:00+00:00",
-            "user": TEST_USER,
-            "usage_id": "test-usage-id",
-            "course_id": "test-course-id",
-            "text": "test note text",
-            "quote": "test note quote",
-            "ranges": [
-                {
-                    "start": "/p[1]",
-                    "end": "/p[1]",
-                    "startOffset": 0,
-                    "endOffset": 10,
+    def tearDown(self):
+        for note_id in note_searcher.all().values_list('id'):
+            get_es().delete(
+                index=settings.ES_INDEXES['default'],
+                doc_type=NoteMappingType.get_mapping_type_name(),
+                id=note_id[0][0]
+            )
+        get_es().indices.refresh()
+
+    @classmethod
+    def setUpClass(cls):
+        get_es().indices.delete(index=settings.ES_INDEXES['default'], ignore=404)
+        get_es().indices.create(
+            index=settings.ES_INDEXES['default'],
+            body={
+                'mappings': {
+                    NoteMappingType.get_mapping_type_name(): NoteMappingType.get_mapping()
                 }
-            ],
-            "permissions": {"read": ["group:__consumer__"]},
-        }
+            },
+        )
+        get_es().indices.refresh()
+        get_es().cluster.health(wait_for_status='yellow')
 
-    def tearDown(self):
-        Annotation.drop_all()
+    @classmethod
+    def tearDownClass(cls):
+        """
+        deletes the test index
+        """
+        get_es().indices.delete(index=settings.ES_INDEXES['default'])
+        get_es().indices.refresh()
 
-    def _create_annotation(self, refresh=True, **kwargs):
+    def _create_annotation(self, **kwargs):
```

Comment:
FYI - [factory boy](http://factoryboy.readthedocs.org/en/latest/) is a great tool for doing what this method and `self.payload` are doing - here's an [example](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/fixtures/discussion.py#L14-L64) of using it to generate dicts from templates in forums bok-choy tests